### PR TITLE
MONDO:0022900 athyreotic congenital hypothyroidism obsolete

### LINKED ISSUE #7357: MONDO:0022900 athyreotic congenital hypothyroidism [Obsolete/Merge ?]
**Mondo term (ID and Label)**
MONDO:0022900
athyreotic congenital hypothyroidism
**Reason for deprecation**

**Suggested term to consider**
MONDO:0019855 athyreosis

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

Note this is one of the terms on #7148; the athyreosis leads to the congenital hypothyroidism, and the GARD page on the obsoletion candidate (https://rarediseases.info.nih.gov/diseases/1610/cretinism-athyreotic) is still 'under development,' makes me wonder whether GARD may not maintain this record going forward.

### PR DESCRIPTION
**Pull request by katiermullen**
closes #7357

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -484878,21 +484878,17 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/1607/crawfur
...
 synonym: "cretinism athyreotic" RELATED DEPRECATED [GARD:0001610]
-xref: GARD:1610 {source="MONDO:GARD"}
-is_a: MONDO:0002254 {source="MONDO:Redundant", source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
-is_a: MONDO:0018612 {source="MONDO:Redundant"} ! congenital hypothyroidism
-relationship: has_characteristic MONDO:0021140 {source="https://orcid.org/0000-0002-4142-7153"} ! congenital
+xref: GARD:1610 {source="MONDO:obsoleteEquivalent"}
+property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeHistoricalDisease"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/4069" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7357" xsd:anyURI
 property_value: IAO:0006012 "2024-06-01" xsd:string
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:484889]: schedule for obsoletion date should be removed

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -484878,21 +484878,17 @@ property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/1607/crawfur
 
 [Term]
 id: MONDO:0022900
-name: athyreotic congenital hypothyroidism
-def: "Congenital hypothyroidism in which fetal deficiency was severe because of complete absence (athyreosis) of the gland. Physical features may include a larger anterior fontanel, persistence of a posterior fontanel, an umbilical hernia, and a large tongue (macroglossia)" [Wikipedia:Congenital_hypothyroidism]
+name: obsolete athyreotic congenital hypothyroidism
+def: "OBSOLETE. Congenital hypothyroidism in which fetal deficiency was severe because of complete absence (athyreosis) of the gland. Physical features may include a larger anterior fontanel, persistence of a posterior fontanel, an umbilical hernia, and a large tongue (macroglossia)" [Wikipedia:Congenital_hypothyroidism]
 comment: Reason of obsoletion: out of scope - MONDO:excludeHistoricalDisease. Term to consider: athyreosis-MONDO:0019855
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:484883]: the comment can be removed since all the information is in annotations. It is not crucial though.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@katiermullen 
There are 2 minor details, which I could have addressed myself. However, since you haven't done a lot of obsoletions in Mondo, I am sending this back to you so it is on your radar.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:484889]: done

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:484883]: @sabrinatoro I was following optional step 11 in the documentation :) I can remove it though if it unnecessary.

Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo]: haha! Indeed, in our document, we say that a comment is helpful, which is why I said "it is not crucial" :-) 
Thank you for following the documentation though !

**Review by sabrinatoro — APPROVED** [review]
Approving and merging!

**Pull request merged**